docs/topics/patterns/token_id_webhook.md

```diff
@@ -0,0 +1,361 @@
+        courseId: 123,
+      });
+      console.log("Message: "  + message);
+      const publishResponse = await topicClient.publish(cacheName, topicName, message);
+      if (publishResponse instanceof TopicPublish.Success) {
+        console.log('Published to the "events" topic in the "cache" cache!');
+      } else if (publishResponse instanceof TopicPublish.Error) {
+        throw new Error(
+          `An error occurred while attempting to publish to the "events" topic in the "cache" cache: ${publishResponse.errorCode()}: ${publishResponse.toString()}`
+        );
+      }
+    } else if (eventsPublishToken instanceof GenerateDisposableToken.Error) {
+      throw new Error(
+        `An error occurred while attempting to call generateApiKey with disposable token scope: ${eventsPublishToken.errorCode()}: ${eventsPublishToken.toString()}`
+      );
+    }
+    return {
+      statusCode: 200,
+      headers: {
+        'Content-Type': 'application/json',
+        'Access-Control-Allow-Origin': '*',
+      },
+      body: '{}',
+    };
+  } catch (err) {
+    console.log(err);
+    return {
+      statusCode: 500,
+      body: JSON.stringify({
+        message: 'An error occurred!',
+      }),
+    };
+  }
+}
+
+async function getAuthClient(): Promise<AuthClient> {
+  const apiKeySecretName = process.env.MOMENTO_API_KEY_SECRET_NAME;
+  if (apiKeySecretName === undefined) {
+    throw new Error("Missing required env var 'MOMENTO_API_KEY_SECRET_NAME");
+  }
+  if (_authClient === undefined) {
+    const momentoApiKey = await getSecret(apiKeySecretName);
+    console.log('Retrieved secret!');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
+    _authClient = new AuthClient({
+      credentialProvider: CredentialProvider.fromString({apiKey: momentoApiKey}),
+    });
+  }
+  return _authClient;
+}
+
+async function getTopicClient(disposableTokenKey: string): Promise<TopicClient> {
+  if (_topicClient === undefined) {
+    console.log('Retrieved secret!');
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
+    _topicClient = new TopicClient({
+      configuration: Configurations.Lambda.latest(),
+      credentialProvider: CredentialProvider.fromString({apiKey: disposableTokenKey}),
+    });
+  }
+
+  return _topicClient;
+}
+
+async function getSecret(secretName: string): Promise<string> {
+  if (!_cachedSecrets.has(secretName)) {
+    const secretResponse = await _secretsClient.send(new GetSecretValueCommand({SecretId: secretName}));
+    if (secretResponse) {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      _cachedSecrets.set(secretName, secretResponse.SecretString!);
+    } else {
+      throw new Error(`Unable to retrieve secret: ${secretName}`);
+    }
+  }
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  return _cachedSecrets.get(secretName)!;
+}
+
+```
+### Conclusion
```

Comment:
Since this isn't a blog post, I wouldn't use `Conclusion` here. Instead, maybe say `Key Takeaways` or some other wording to highlight the key points

Reply:
On second thought, maybe we boil it down to a couple of small bullet points then add a `Keep Reading` section that uses the ReferenceCard widget to navigate people to other content, like the multi language chat app, security post, etc... [Example](https://github.com/momentohq/public-dev-docs/blob/main/docs/topics/webhooks/overview.mdx#L110)